src/pages/index.tsx

```diff
@@ -156,15 +160,11 @@ export default function HomePage({ schedule }) {
           >
             {userLocation.lat && userLocation.lon ? 'Lokasi Ditemukan' : 'Dapatkan Lokasi Anda'}
           </Button>
-          <Select maxW={['auto', '2xs']} onChange={e => setSearchBy(e.target.value)} value={searchBy}>
-            <option value="kecamatan">Kecamatan</option>
-            <option value="kelurahan">Kelurahan</option>
-          </Select>
           <Input
             flexGrow={1}
             fontSize={[14, 16]}
             onChange={e => setSearchKeyword(e.target.value)}
-            placeholder="cari kecamatan/kelurahan"
+            placeholder="cari nama lokasi vaksinasi/wilayah/kecamatan/kelurahan"
```

Comment:
I think wilayah add confusion, we can just use lokasi / kecamatan / kelurahan

Also reorder the list so kecamatan & kelurahan go first, as it's more common

```suggestion
            placeholder="cari kecamatan / kelurahan / nama lokasi vaksinasi"
```